Framework/Algorithms/src/ModeratorTzeroLinear.cpp

```diff
@@ -144,17 +144,17 @@ void ModeratorTzeroLinear::exec() {
     {
       const double scaling = L_i / (L_i + m_gradient);
       const double offset = (1 - scaling) * t_f - scaling * m_intercept;
-      const MantidVec &inbins = inputWS->readX(i);
-      MantidVec &outbins = outputWS->dataX(i);
+      auto &inbins = inputWS->x(i);
+      auto &outbins = outputWS->mutableX(i);
       for (unsigned int j = 0; j < inbins.size(); j++) {
         outbins[j] = scaling * inbins[j] + offset;
```

Comment:
`outbins = scaling * inbins + offset` would work, but is in theory slightly slower. Would be interesting to test the performance?